integration/src/main/java/scripts/testscripts/RemoveUser.java

```diff
@@ -59,6 +59,16 @@ protected void doSetup(List<TestUserSpecification> testUsers, WorkspaceApi owner
         sharedResourceUser.userEmail,
         "The two test users are distinct");
 
+    // Create a group
+    groupName = TestUtils.appendRandomNumber("groupremovetest");
+    samGroupApi = SamClientUtils.samGroupApi(testUsers.get(1), server);
+    SamRetry.retry(() -> samGroupApi.postGroup(groupName, null));
```

Comment:
nit it's nice to add a comment with parameter name, since `null` isn't clear

```
SamRetry.retry(() -> samGroupApi.postGroup(groupName, /*body=*/ null));
```